apps/site/assets/ts/helpers/service.ts

```diff
@@ -74,7 +74,8 @@ export const isCurrentValidService = (
   }
 
   // check on valid_days
-  if (!isOnValidDay(service, currentDate)) {
+  // disregard if empty valid_days value
+  if (service.valid_days.length > 0 && !isOnValidDay(service, currentDate)) {
     return false;
   }
```

Comment:
Minor:
Refactor to just return the boolean (I think I applied DeMorgans Law correctly)
```suggestion
  return (service.valid_days.length <= 0 || isOnValidDay(service, currentDate));
```

Reply:
Thanks. I ended up rethinking this and going with a different approach :weary: